packages/core/src/controllers/store.ts

```diff
@@ -87,6 +87,14 @@ export class Store<Key, Data extends StoreStruct> extends IStore<Key, Data> {
     return value;
   };
 
+  public getAll: IStore<Key, Data>["getAll"] = filter => {
+    if (!filter) return this.values;
+
+    return this.values.filter(value =>
+      Object.keys(filter).every(key => value[key] === filter[key]),
```

Comment:
Not a big concern right now, but this will only work with JS primitive types, not if users want to use object/array structures to filter by (e.g.  filtering pairings by `peerMetadata`)

Reply:
Yeah exactly `isEqual` my bad, sweet 👍